src/hyperloglog.c

```diff
@@ -1041,6 +1045,121 @@ int hllAdd(robj *o, unsigned char *ele, size_t elesize) {
     }
 }
 
+#if defined(HAVE_AVX2)
+/* Optimized for the default configuration: 16384 registers of 6 bits each. */
+ATTRIBUTE_TARGET_AVX2
+void hllMergeDense_AVX2(uint8_t *reg_raw, const uint8_t *reg_dense) {
```

Comment:
in redis, we usually don't use `_` in function name
```suggestion
void hllMergeDenseAVX2(uint8_t *reg_raw, const uint8_t *reg_dense) {
```